compiler/aarch64/codegen/UnaryEvaluator.cpp

```diff
@@ -81,17 +81,36 @@ TR::Register *OMR::ARM64::TreeEvaluator::lnegEvaluator(TR::Node *node, TR::CodeG
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+static TR::Register *commonAbsEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Register *trgReg = cg->allocateRegister();
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Register *srcReg = cg->evaluate(firstChild);
```

Comment:
Could this be a clobberEvaluate to save a register?  I think you still need a temp register, but the target and source could potentially be shared.